packages/react/src/Render.js

```diff
@@ -23,6 +23,17 @@ export default class Render extends React.Component<RenderComponentProps> {
     let render = this.props.render || this.context.linguiDefaultRender
 
     if (render === null || render === undefined) {
+      if (process.env.NODE_ENV !== "production" && typeof value === "string") {
+        if (/^15\./.test(React.version)) {
```

Comment:
I wanted to make this just one `if` statement but eslint got mad and I have to figure out how to run prettier to calm it down.